packages/core/src/components/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { makeStyles, Typography, Grid } from '@material-ui/core';
+import { EmptyStateImage } from './EmptyStateImage';
+import Background from './assets/Background.svg';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    backgroundColor: theme.palette.background.default,
+    padding: theme.spacing(10, 0, 0, 0),
+  },
+  action: {
+    marginTop: theme.spacing(2),
+  },
+  imgPlacholder: {
+    position: 'relative',
+  },
+  backgroundImage: {
+    position: 'absolute',
+    width: '100%',
+  },
+}));
+
+type Props = {
+  title: string;
+  description?: string;
+  type: 'missingAnnotation' | 'noInformation' | 'createComponent' | 'noBuild';
```

Comment:
Wonder if we can make these less specific, perhaps `'field' | 'info' | 'content' | 'data'` or something like that?

Thinking about reading the API it might also be nicer to invert the property meaning here a bit by renaming to `missing`, thinking

```tsx
<EmptyState missing='data'/>
```

Reply:
Great idea